src/python/grpcio/grpc/experimental/aio/_call.py

```diff
@@ -221,63 +229,24 @@ def __str__(self) -> str:
         return self._repr()
 
 
-class UnaryUnaryCall(Call, _base_call.UnaryUnaryCall):
-    """Object for managing unary-unary RPC calls.
+class _UnaryResponseMixin(Call):
+    _call_finisher: asyncio.Task
```

Comment:
Semantically I would prefer something like `_call_response` or `_response_call` or something like that.